Patch by Robert Scheck <robert@fedoraproject.org> for mutt <= 1.5.18, which
adds the ability to link mutt against db <= 4.7 rather the original <= 4.6.
This patch was sent upstream and hopefully should be included into the next
release of mutt.

--- mutt-1.5.18/configure		2008-07-12 20:34:15.000000000 +0200
+++ mutt-1.5.18/configure.db47		2008-07-12 20:39:52.000000000 +0200
@@ -17397,7 +17397,7 @@
                 bdbpfx="$bdbpfx $d/$v"
             done
         done
-        BDB_VERSIONS="db-4 db4 db-4.6 db4.6 db46 db-4.5 db4.5 db45 db-4.4 db4.4 db44 db-4.3 db4.3 db43 db-4.2 db4.2 db42 db-4.1 db4.1 db41 db ''"
+        BDB_VERSIONS="db-4 db4 db-4.7 db4.7 db-4.6 db4.6 db46 db-4.5 db4.5 db45 db-4.4 db4.4 db44 db-4.3 db4.3 db43 db-4.2 db4.2 db42 db-4.1 db4.1 db41 db ''"
         { $as_echo "$as_me:$LINENO: checking for BerkeleyDB > 4.0" >&5
 $as_echo_n "checking for BerkeleyDB > 4.0... " >&6; }
         for d in $bdbpfx; do
